strings/tips/marine.txt

```diff
@@ -1,10 +1,10 @@
 As a human, you can do almost anything if you put your mind to it; it just takes time.
 While medics and doctors are best at it, anyone can use a defibrillator and operate the body scanner and autodoc medical system.
 Anyone can make razor wire with four metal rods and barbed wire from metal sheets.
-Using a multitool on a lasgun battery will turn it into an improvised explosive with a random timer.
-In a pinch, you can use lasgun batteries to power APCs and sentry guns.
-Lasgun batteries can be recharged in APCs and power cell rechargers, if the generators are on.
-As a marine, if you aren't a PFC, then you have a special room in prep with special gear vendors.
+Using a multitool on a Terra Experimental standard battery will turn it into an improvised explosive with a random timer.
+In a pinch, you can use Terra Experimental standard batteries to power APCs and sentry guns.
```

Comment:
Sentry guns should no longer use batteries save for actual laser ammo rather than power